src/workers/newNotificationPush.ts

```diff
@@ -20,7 +20,7 @@ const worker: Worker = {
           notificationId: notification.id,
           order: 0,
         });
-        await sendPushNotification(notification, avatar);
+        await sendPushNotification([notification.userId], notification, avatar);
```

Comment:
Did you not change this to userIds as in plurar and multiple?
Or does the main thing already split them (can't really recall)

Reply:
This is worker v1 so user id is still embedded in the notification object